models/ed/R/read.restart.ED2.R

```diff
@@ -0,0 +1,100 @@
+# TODO: Change inheritParams to template (also need to create template)
+
+#' @title State data assimilation read-restart for ED2
+#'
+#' @author Alexey Shiklomanov
+#' @inheritParams PEcAn.SIPNET::read.restart.SIPNET
+#' @examples
+#' \dontrun{
```

Comment:
@mdietze I agree that this should be referencing a template package and not SIPNET -- this was a temporary solution because we don't have template `read.` and `write.` restart functions yet. I will draft blank versions as part of this PR -- that won't take long.

I will push back on the rest though. `inheritParams` is actually pretty smart -- what it does is match only existing function parameters that have not yet been documented, so that if there are any SIPNET-specific parameters that are not arguments to `write.restart.ED2`, they will be ignored ([more information](http://r-pkgs.had.co.nz/man.html#dry2). Given that, I think it's actually bad practice _not_ to use `inheritParams` for any standard `read.` and `write.restart` function parameters (e.g. `outdir`, `runid`, `settings`), since we will want these to mean the same thing in every context. For that matter, as we're revising PEcAn code, I think we should replace a lot of our parameter descriptions with `inheritParams` to enforce consistency and encourage parameters to be well documented once rather than haphazardly or inconsistently documented (or not documented at all) in many different places.

As far as human readability, if you actually look at the generated `.Rd` files, you'll see that the full parameter description is transferred from the SIPNET version -- there are no links to follow. The only place where this arguably reduces human readability is in the source code itself, but if people are looking at the sources anyway, it's easy enough for them to trace this back to the right package.